### PR TITLE
Use longtable for the features table

### PR DESCRIPTION
**Pull request by asmeurer**
This way, it breaks across pages. See
https://tex.stackexchange.com/questions/26462/make-a-table-span-multiple-pages.

[Build pdf](https://github.com/isuruf-bot/sympy-paper/blob/pdfs/paper-69.pdf)

### CODE REVIEW

File: features.tex [hunk 1]
```diff
@@ -65,8 +63,9 @@
     Legendre polynomial, Hermite polynomial, Laguerre polynomial, and
     spherical harmonic functions.\\
 \hline
-\end{tabular}
-\end{table}
+\caption{SymPy Features and Descriptions}
+\label{features-table}
```
Comment (scopatz) [inline features.tex]: Table captions typically appear on the top of tables, not on the bottom, right?

**Comment by scopatz** [comment]
Looks good, except for the caption location.

File: features.tex (hunk 1 above)
Comment (asmeurer) [inline features.tex]: I can't figure out how to move it.

Comment (scopatz) [inline features.tex]: Oh weird ok.  There is probably some longtable option.

Comment (scopatz) [inline features.tex]: Ok nm. it looks like it will render correctly

Comment (asmeurer) [inline features.tex]: It is under the table this way (previously it was over).

Comment (scopatz) [inline features.tex]: Hmm Section 3 of the docs probably explain how to change this: http://ctan.mirrors.hoobly.com/macros/latex/required/tools/longtable.pdf

Have to run to class right now, though.

Comment (asmeurer) [inline features.tex]: Figured it out (you have to put `\\` at the end of the line).

**Comment by asmeurer** [comment]
There is an issue here. The bottom of the table has some extra lines sticking out of it.
screen shot 2016-04-20 at 12 59 35 pm

Any idea what's causing that?

**Comment by certik** [comment]
Regarding the extra lines, perhaps try to move the `\label` line from the bottom to the top of the table or something like that.

**Comment by asmeurer** [comment]
Good catch. I didn't notice that the label should go inside the caption. Should be ready to go now.

**Comment by asmeurer** [comment]
To whoever included booktabs (@moorepants?), should we reformat the table to use that?

**Comment by moorepants** [comment]
I think so. The booktabs look tons nicer.

**Comment by asmeurer** [comment]
Going to merge this now to keep the PR count down. But let's definitely figure out how to format the tables.

**Pull request merged**